lib/signaling/v2/room.js

```diff
@@ -241,8 +265,8 @@ class RoomV2 extends RoomSignaling {
   /**
    * @private
    */
-  _getTrackReceiver(id) {
-    return this._getOrCreateTrackReceiverDeferred(id).promise.then(trackReceiver => {
+  _getTrackReceiver(id, idOrMid = 'id') {
```

Comment:
similar to earlier suggestion
```suggestion
  _getTrackReceiver(propertyValue, propertyName = 'id') {
```

Reply:
See my earlier [response](https://github.com/twilio/twilio-video.js/pull/1732/files#r828499850).